⏣ Version 1.2

tooxo opened the pull request.

Commit:
Added Test Coveralls

Commit:
Modify Coveralls

Commit:
Multiple Improvements

pep8speaks commented:
Hello @tooxo! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`discord/discord_music.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py):

> [Line 301:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L301): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 302:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L302): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 307:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L307): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 308:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L308): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 329:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L329): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 389:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L389): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 530:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L530): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 579:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L579): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 580:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L580): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 659:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L659): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 891:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L891): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 892:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L892): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1179:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1179): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1180:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1180): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1181:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1181): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1182:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1182): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1215:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1215): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1216:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1216): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1217:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1217): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1218:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1218): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1340:52](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1340): [E203](https://duckduckgo.com/?q=pep8%20E203) whitespace before ':'
> [Line 1385:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1385): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1493:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1493): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 1494:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/discord_music.py#L1494): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator









* In the file [`discord/extractors/spotify.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/extractors/spotify.py):

> [Line 80:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/extractors/spotify.py#L80): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 81:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/extractors/spotify.py#L81): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 93:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/extractors/spotify.py#L93): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 94:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/extractors/spotify.py#L94): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator

* In the file [`discord/extractors/youtube.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/extractors/youtube.py):

> [Line 18:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/extractors/youtube.py#L18): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 19:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/extractors/youtube.py#L19): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator

* In the file [`discord/now_playing_message.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/now_playing_message.py):

> [Line 47:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/now_playing_message.py#L47): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 50:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/now_playing_message.py#L50): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 61:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/now_playing_message.py#L61): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 62:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/now_playing_message.py#L62): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 63:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/now_playing_message.py#L63): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 71:21](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/discord/now_playing_message.py#L71): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator









* In the file [`web/server.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/web/server.py):

> [Line 21:9](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/web/server.py#L21): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator



* In the file [`youtube/node/node.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py):

> [Line 24:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L24): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 25:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L25): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 76:29](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L76): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 77:29](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L77): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 78:29](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L78): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 79:29](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L79): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 80:29](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L80): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 128:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L128): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 129:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L129): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 174:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L174): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 175:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L175): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 278:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L278): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 279:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L279): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 280:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L280): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 281:25](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L281): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 288:13](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/node/node.py#L288): [E722](https://duckduckgo.com/?q=pep8%20E722) do not use bare 'except'

* In the file [`youtube/parent.py`](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py):

> [Line 89:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L89): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 90:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L90): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 91:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L91): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 92:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L92): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 93:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L93): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 94:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L94): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 95:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L95): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 113:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L113): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 114:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L114): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 115:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L115): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 116:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L116): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 150:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L150): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 151:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L151): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 152:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L152): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator
> [Line 153:17](https://github.com/tooxo/Geiler-Musik-Bot/blob/bc6f8c37e06fdc0de4f07b1650176f3f08f60f29/youtube/parent.py#L153): [W503](https://duckduckgo.com/?q=pep8%20W503) line break before binary operator



##### Comment last updated at 2019-11-28 22:00:10 UTC

Commit:
Reformatted with Black

tooxo committed:
Merge branch 'master' into V1_2

Commit:
Corrected to PEP8 Standards

Commit:
Merge remote-tracking branch 'origin/V1_2' into V1_2

Commit:
Various Changes

Commit:
Added Basic Tests

Commit:
Test Script Fixed

Commit:
Test Script Fixed II

Commit:
Tests Fixed III

Commit:
Tests Fixed IV

Commit:
Tests Fixed V

Commit:
Tests Fixed VI

Commit:
Tests Fixed VII

Commit:
Tests Fixed VIII

Commit:
Tests Fixed IX

Commit:
Tests Fixed X (this time for real)

Commit:
PEP8 Updated

Commit:
Bugfixes +1 Test

Commit:
Added GoogleVideo Manifest Extraction

Commit:
Bugfixes & General Improvements

Commit:
Added Black Formatting

Commit:
Multiple Improvements & Test Fix

Commit:
Greater Delays in testing.

Commit:
.

Commit:
Separated Installation from tests.

Commit:
.

Commit:
.

Commit:
Fixed Connecting Bug, General Improvements.

Commit:
multiple changes

externalized database
externalized youtube gathering
smaller build size
fixed messaging
switched to bjoern

Commit:
code refactoring

Commit:
longer test time

Commit:
updated build vars

Commit:
debug statements

Commit:
debug statements

Commit:
fixed blocking bug

Commit:
make mongo disable-able

Commit:
fixed stuff
